tests/path_gateway_ipns_test.go

```diff
@@ -7,6 +7,74 @@ import (
 	. "github.com/ipfs/gateway-conformance/tooling/test"
 )
 
+var (
+	// See fixtures/ipns_records/README.md for information
+	ipnsV1                = "k51qzi5uqu5dm4tm0wt8srkg9h9suud4wuiwjimndrkydqm81cqtlb5ak6p7ku"
+	ipnsV1V2BrokenValueV1 = "k51qzi5uqu5dlmit2tuwdvnx4sbnyqgmvbxftl0eo3f33wwtb9gr7yozae9kpw"
+	ipnsV1V2BrokenSigV2   = "k51qzi5uqu5diamp7qnnvs1p1gzmku3eijkeijs3418j23j077zrkok63xdm8c"
+
+	ipnsV1V2BrokenSigV1     = "k51qzi5uqu5dilgf7gorsh9vcqqq4myo6jd4zmqkuy9pxyxi5fua3uf7axph4y"
+	bodyIPNSV1V2BrokenSigV1 = []byte("v1+v2 with broken signature v1")
+
+	ipnsV1V2     = "k51qzi5uqu5dlkw8pxuw9qmqayfdeh4kfebhmreauqdc6a7c3y7d5i9fi8mk9w"
+	bodyIPNSV1V2 = []byte("v1+v2 record")
+	cidIPNSV1V2  = "bafkqaddwgevxmmraojswg33smq"
+
+	ipnsV2     = "k51qzi5uqu5dit2ku9mutlfgwyz8u730on38kd10m97m36bjt66my99hb6103f"
+	bodyIPNSV2 = []byte("v2-only record")
+	cidIPNSV2  = "bafkqadtwgiww63tmpeqhezldn5zgi"
+)
```

Comment:
Initially, the goal was to move around car files so that we could store them elsewhere (in the specs repo for example). We'd reduce dependencies on hardcoded values as we had in sharness.

That's why we wrote fixtures loaders like:

```go
	fixture := car.MustOpenUnixfsCar("ipns_records/fixtures.car")
	file := fixture.MustGetRoot()
	fileCID := file.Cid()

	ipns := MustOpenIPNSRecordWithKey("ipns_records/k51qzi5uqu5dh71qgwangrt6r0nd4094i88nsady6qgd1dhjcyfsaqmpp143ab.ipns-record")
	ipnsName := ipns.Key()
```

or

https://github.com/ipfs/gateway-conformance/blob/fd7e8c954ed7f38e3a01ced47f23929b7108cfed/tests/subdomain_gateway_ipfs_test.go#L108-L116


- Eventually, we'd use assets (car files) coming from the specs, and the test-suite would update "declaratively" and throw an error as soon as you'd load the test (because the file would be missing from a car file for example).
- We also discussed having ipns records in car files, which meant we could remove hardcoding keys in filenames.


To me this seems to be a step back, so I'd rather make sure we're making this change consciously.

Reply:
@laurentsenta I'm making this very consciously. Most of these records are invalid, and therefore `ipns.UnmarshalRecord` will fail. So we can't load them like that.

Reply:
These could use the regular API, right? 

```go
	ipnsV1V2     = "k51qzi5uqu5dlkw8pxuw9qmqayfdeh4kfebhmreauqdc6a7c3y7d5i9fi8mk9w"
	bodyIPNSV1V2 = []byte("v1+v2 record")
	cidIPNSV1V2  = "bafkqaddwgevxmmraojswg33smq"

	ipnsV2     = "k51qzi5uqu5dit2ku9mutlfgwyz8u730on38kd10m97m36bjt66my99hb6103f"
	bodyIPNSV2 = []byte("v2-only record")
	cidIPNSV2  = "bafkqadtwgiww63tmpeqhezldn5zgi"
```

Reply:
@laurentsenta, yes, I can update the ones that can use the regular API, I'll update them

Reply:
@laurentsenta updated.